core/pom.xml

```diff
@@ -67,6 +67,11 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.2.12</version>
```

Comment:
The version should go in the parent POM's dependencyManagement section. The child POMs should omit the version.